runtime/caml/startup.h

```diff
@@ -21,8 +21,6 @@
 #include "mlvalues.h"
 #include "exec.h"
 
-CAMLextern void caml_main(char_os **argv);
```

Comment:
Note to self and other readers: this declaration was protected by CAML_INTERNALS; removing it should not cause issues with external C code.